keras_cv/layers/preprocessing/cut_mix.py

```diff
@@ -0,0 +1,172 @@
+import tensorflow as tf
+import tensorflow.keras as keras
+import tensorflow.keras.layers as layers
+from tensorflow.python.platform import tf_logging as logging
+
+
+class CutMix(layers.Layer):
+    """CutMix implements the CutMix data augmentation technique.
+
+    Args:
+        rate: Float between 0 and 1.  The fraction of samples to augment.
+        alpha: Float between 0 and 1.  Inverse scale parameter for the gamma distribution.
+            This controls the shape of the distribution from which the smoothing values are 
+            sampled.  Defaults 1.0, which is a recommended value when training an imagenet1k
+            classification model.
+        label_smoothing: coefficient used in label smoothing.  Default 0.0.
+    References:
+        https://arxiv.org/abs/1905.04899.
```

Comment:
Use markdown link (since we will use the docstrings to generate rendered docs pages in the future)

Reply:
Done